sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeCustomFormsOperation.cs

```diff
@@ -174,9 +174,10 @@ private static RecognizedFormCollection ConvertToRecognizedForms(AnalyzeResult_i
         private static RecognizedFormCollection ConvertUnsupervisedResult(AnalyzeResult_internal analyzeResult)
         {
             List<RecognizedForm> forms = new List<RecognizedForm>();
-            foreach (var pageResult in analyzeResult.PageResults)
+            for (int pageIndex = 0; pageIndex < analyzeResult.PageResults.Count; pageIndex++)
             {
-                forms.Add(new RecognizedForm(pageResult, analyzeResult.ReadResults));
+                var pageResult = analyzeResult.PageResults[pageIndex];
```

Comment:
do we use this pageResult?

Reply:
No, we don't. I've changed this file a lot of times for this fix, so I forgot this variable there.